README.md

```diff
@@ -38,6 +38,8 @@ $ sudo make install # optional
 $ sudo ldconfig     # optional
 ```
 
+In general, you should use the install.sh scripts for Linux or MacOS rather than building from source.
```

Comment:
Both techniques build from source (and from the current branch). The distinction is that the install.sh is simpler to execute (and works around certain obscure difficulties).

Reply:
Perhaps describing my first time user experience as a n00b would be helpful... I've been in a similar boat where it seems obvious when you're close to the project:
- I read the build instructions first and thought that was the recommended way to do things.  I hit a stumbling block on a dependency, grabbed that, and then worked my way backwards until I found the original dependency (libbitcoin itself, which in retrospect seems obvious).
- Once built (v4), my understanding is that most community servers are still running v3, but they're still the default settings in the config file. I think this is why I was having issues fetching balances using bx fetch-balance.
- I realized my error and tried using the install.sh script on an Ubuntu 16.04 EC2 image. I hit a stumbling block with something in the makefile and it would abort partway through.
- Perplexed, I used homebrew to install BX on my Mac, which installed v3.4. This worked correctly and was able to fetch balances with no issue.

This is not a complaint, I just want to be helpful. I definitely appreciate the insane amount of work this project has probably required thus far - it's a really impressive set of tools.